hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1439,6 +1439,72 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+usb_max_drift=1", "+usb_fast_sof=1"]
       reseed: 1
+    }
+        {
+      name: chip_sw_clkmgr_jitter_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:clkmgr_jitter_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_flash_ctrl_ops_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:flash_ctrl_ops_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=14_000_000", "+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_flash_ctrl_access_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:flash_ctrl_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_otbn_ecdsa_op_irq_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:otbn_ecdsa_op_irq_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=33_000_000", "+rng_srate_value=30", "+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+      run_timeout_mins: 1000
+    }
+    {
+      name: chip_sw_aes_enc_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:aes_smoketest:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=26_000_000", "+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_hmac_enc_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:hmac_enc_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_keymgr_key_derivation_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_keymgr_key_derivation_vseq
+      sw_images: ["//sw/device/tests/sim_dv:keymgr_key_derivation_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=20_000_000", "+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_kmac_mode_kmac_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:kmac_mode_kmac_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+en_jitter=1", "+cal_sys_clk_70mhz=1"]
+    }
+    {
+      name: chip_sw_sram_ctrl_main_scrambled_access_jitter_en_reduced_freq
+      uvm_test_seq: chip_sw_sram_ctrl_scrambled_access_vseq
+      sw_images: ["//sw/device/tests/sim_dv:sram_ctrl_main_scrambled_access_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+mem_sel=main",
+                 "+sw_test_timeout_ns=12_000_000",
+                 "+en_jitter=1", "+en_scb_tl_err_chk=0", "+cal_sys_clk_70mhz=1"]
```

Comment:
there's also a `flash_init` test, do you want to put that in here?

Reply:
Added flash_init test with reduced frequency as well

Reply:
thanks for adding it. this may also have to be added to `jitter_reduced_freq`, see other comment.

Reply:
Flash_Init was added to the jitter_reduced_freq list.
But please note the jitter is disabled in this test.
When running the test with reduced frequency the test pass, when trying to enable the jitter the test fail, with the following error:
UVM_FATAL @  10.817875 us: (chip_env_pkg.sv:193) [sw_symbol_get_addr_size] Check failed (ret) Failed to read line from "kJitterEnabled.dat"

Reply:
@sriyerg @weicaiyang do you have an idea why this might be happening?

Reply:
it's because flash_init is running with `sw_test_mode_common`, which means it uses mask rom instead of test rom.
We can only set `en_jitter` for test rom.

I vaguely remember mask rom always enables jitter, but I'm not quite sure if it's correct. @timothytrippel may know better.

Reply:
The production (mask) ROM sets the enablement state of the clock jitter based on its OTP configuration (see [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/rom_start.S;drc=ef9cac1bf078dfc32237a4049db2ee34b7787897;l=211)). The default OTP image used for all tests is the RMA image generated by dvsim [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/chip_sim_cfg.hjson;drc=d38e18f6df5f20495ba242d967fe09f304104e40;l=243), which [disables](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/otp_ctrl/data/otp_ctrl_img_creator_sw_cfg.hjson;drc=dac6553f2176d49ec6c18024bac4812d5847bb91;l=61) clock jitter. As @weicaiyang mentions, only the test ROM has the ability to enable clock jitter by backdoor overwriting the `kJitterEnabled` [symbol](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_rom/test_rom.c;drc=517edba6abe68b919742d7c1d0a687361b9fe32b;l=164) via the plusarg above.

However, if you would like to enable jitter and use the ROM, its OTP setting can be overridden by feeding the test a different OTP image built by bazel (assuming there is a Bazel target to build the OTP image you would like; see [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/e2e/BUILD;drc=d0eee8d60f14bb47fbf48c5e0f00a04dee01b8e1;l=801) for an example). See lines [35](https://github.com/lowRISC/opentitan/blob/d38e18f6df5f20495ba242d967fe09f304104e40/hw/top_earlgrey/dv/chip_rom_tests.hjson#L35) and [40](https://github.com/lowRISC/opentitan/blob/d38e18f6df5f20495ba242d967fe09f304104e40/hw/top_earlgrey/dv/chip_rom_tests.hjson#L40) in `chip_rom_tests.hjson` for an example.

@alphan can confirm.

Reply:
is it disabled in DV sims due to this entry here?
https://github.com/lowRISC/opentitan/blob/52d1ab6acec59b9c90455135b12766220a0ef54b/hw/ip/otp_ctrl/data/otp_ctrl_img_creator_sw_cfg.hjson#L59-L62
@moidx does it make sense to enable jitter for all DV tests that use the real mask ROM?

Reply:
No objections from my end if we want to do this to unblock this PR.

Since until this point we have been tracking jitter tests explicitly, does it make sense to make this configurable in the test bench when running with `rom` in the long run?

Reply:
Interestingly, the SV sequence already overwrites the OTP location via backdoor if the plusarg `+en_jitter=1` is set:
https://github.com/lowRISC/opentitan/blob/c1e8a819c8e02f2608c6fa797bd769adf87bac76/hw/top_earlgrey/dv/env/seq_lib/chip_sw_base_vseq.sv#L118-L134

The problem here is that `sw_symbol_backdoor_overwrite` with the real ROM since the symbol does not exist in that case. If we can figure out from SV whether we use the real ROM or the test ROM, this would be an easy SV fix.

Alternatively, we can just use a different argument name or value, e.g. `+en_jitter=ROM` vs. `+en_jitter=TEST_ROM`.

Reply:
If we will enable jitter in the production chip, perhaps we should by default enable it in mask rom

Reply:
> Interestingly, the SV sequence already overwrites the OTP location via backdoor if the plusarg `+en_jitter=1` is set:
> 
> https://github.com/lowRISC/opentitan/blob/c1e8a819c8e02f2608c6fa797bd769adf87bac76/hw/top_earlgrey/dv/env/seq_lib/chip_sw_base_vseq.sv#L118-L134
> 
> The problem here is that `sw_symbol_backdoor_overwrite` with the real ROM since the symbol does not exist in that case. If we can figure out from SV whether we use the real ROM or the test ROM, this would be an easy SV fix.
> 
> Alternatively, we can just use a different argument name or value, e.g. `+en_jitter=ROM` vs. `+en_jitter=TEST_ROM`.

wonder if we can completed remove `kJitterEnabled` and use `cfg.mem_bkdr_util_h` to control the jitter enable.

Reply:
Yeah, I think the issue is that the ROM assembly may override this bit based on the OTP configuration value. Lets change the OTP default value in this PR.  I don't think we need to solve the production ROM configuration in this PR. I only ask to make it configurable in case we need to disable it in production so that we can have a way to reproduce issues in simulation, but in as-is we'll be able to reproduce issues in that case by changing the OTP config vals.

Reply:
I guess we could keep this switch if we make sure that the test ROM loads this switch from OTP also? That way we would not need the SW symbol `kJitterEnabled` anymore...

Reply:
Anyways, for the sake of this PR, I believe we can move forward and merge it, and make the OTP alignments in a follow-up PR...

Reply:
> The production (mask) ROM sets the enablement state of the clock jitter based on its OTP configuration (see [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/rom_start.S;drc=ef9cac1bf078dfc32237a4049db2ee34b7787897;l=211)). The default OTP image used for all tests is the RMA image generated by dvsim [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/chip_sim_cfg.hjson;drc=d38e18f6df5f20495ba242d967fe09f304104e40;l=243), which [disables](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/otp_ctrl/data/otp_ctrl_img_creator_sw_cfg.hjson;drc=dac6553f2176d49ec6c18024bac4812d5847bb91;l=61) clock jitter. As @weicaiyang mentions, only the test ROM has the ability to enable clock jitter by backdoor overwriting the `kJitterEnabled` [symbol](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_rom/test_rom.c;drc=517edba6abe68b919742d7c1d0a687361b9fe32b;l=164) via the plusarg above.
> 
> However, if you would like to enable jitter and use the ROM, its OTP setting can be overridden by feeding the test a different OTP image built by bazel (assuming there is a Bazel target to build the OTP image you would like; see [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/e2e/BUILD;drc=d0eee8d60f14bb47fbf48c5e0f00a04dee01b8e1;l=801) for an example). See lines [35](https://github.com/lowRISC/opentitan/blob/d38e18f6df5f20495ba242d967fe09f304104e40/hw/top_earlgrey/dv/chip_rom_tests.hjson#L35) and [40](https://github.com/lowRISC/opentitan/blob/d38e18f6df5f20495ba242d967fe09f304104e40/hw/top_earlgrey/dv/chip_rom_tests.hjson#L40) in `chip_rom_tests.hjson` for an example.
> 
> @alphan can confirm.

Confirmed. Also, we run `rom_e2e_asm_init` which covers this case in the CI (see #14493).

Reply:
Great, thanks @alphan!